.env_SAMPLE

```diff
@@ -108,9 +122,9 @@ export WORDPRESS=http://www.consumerfinance.gov
 # TESTING (optional) - for handling testing of the site.
 ########################################################
 
-#export HTTP_HOST=localhost
-#export HTTP_PORT=8000
-#export SELENIUM_URL=http://localhost:4444/wd/hub
+#export SITE_DESC=cfgov
+#export TEST_HTTP_HOST=localhost
+#export TEST_HTTP_PORT=8000
```

Comment:
> Did you specifically want to change the behavior so that you don't have to do anything special to .env in order to run the tests?

Yeah, I wanted it so that certain defaults are set out of the box and everything should work. There's no reason to force editing of `.env` unless the user wants to change to a special setup (such as running the server on a different port).

I realized the other reason I put the defaults in `.env` was so they're set in one file. If you search the changed files for `8000` you'll see that that is set as a default value in several places across the codebase, which should really only happen in one spot.

Still mulling how to approach this. Throwing an error in `environment.js` would mean the error wouldn't be thrown when the environment vars are set in `.env`.